SoftLayer/CLI/hardware/create.py

```diff
@@ -47,6 +47,13 @@
 @click.option('--no-public',
               is_flag=True,
               help="Private network only")
+@click.option('--vlan-private',
```

Comment:
Is that an IBM VPC equivalent?

Reply:
Yes. For some reason, their API requires both the VLAN ID plus the subnet ID, even though the VLAN ID should be derivable.